modules/backup-etcd.adoc

```diff
@@ -40,7 +40,25 @@ $ oc debug --as-root node/<node_name>
 sh-4.4# chroot /host
 ----
 
-. If the cluster-wide proxy is enabled, be sure that you have exported the `NO_PROXY`, `HTTP_PROXY`, and `HTTPS_PROXY` environment variables.
+. If the cluster-wide proxy is enabled, ensure that you exported the `NO_PROXY`, `HTTP_PROXY`, and `HTTPS_PROXY` environment variables.
++
+.. To export the `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` environment variables, run the following commands:
+
```

Comment:
I think the formatting issues will be resolved if you remove this line. For example, the text should look like this:

```
.. To export the `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` environment variables, run the following commands:
+
[source,terminal]
----
$ export HTTP_PROXY=http://<your_proxy.example.com>:8080
----
+
[source,terminal]
----
$ export HTTPS_PROXY=https://<your_proxy.example.com>:8080
----
+
[source,terminal]
----
$ export NO_PROXY=<example.com>
----